src/Test/EventListenerIntrospectionTrait.php

```diff
@@ -9,7 +9,7 @@
 
 namespace Zend\EventManager\Test;
 
-use PHPUnit_Framework_Assert as Assert;
+use PHPUnit\Framework\Assert;
```

Comment:
before the `Assert` class it was with the `TestCase` class

https://github.com/zendframework/zend-eventmanager/pull/19/commits/be75ec1c3ddafaeb56f99c654a8968ec8463c041

from my site i see the `Assets` class as better solution, with `$this` it could be possible that it break smth in other tests if someone not use the `TestCase` as parent-class. also we have a autocomplete and no warning that the method not exist.